docs/event-serializing.md

```diff
@@ -0,0 +1,139 @@
+---
+id: event-serializing
+title: Event Serializing
+sidebar_label: Event Serializing
+---
+
+We support [Apache Avro](https://avro.apache.org/) and standard JSON (de)serialization of events for producer as well as for consumer. This is working together with [Cloud Events specification](https://github.com/cloudevents/spec) (versions `0.1` and `0.2`). To make both of these worlds work we are utilizing Kafka headers introduced in version `0.11`. As Cloud Events specify there are 2 modes how to make this combination work. Discussion about this topic can be followed on [Github](https://github.com/cloudevents/spec/pull/337/files).
+
+**Structured content mode**
+
+This mode should be used when JSON format is desired, thus no further serialization and deserialization such as Apache Avro.
```

Comment:
not sure I get this 😕 

1. if you need json, you can use both modes, the difference is just that in structured mode the context attributes are included in the json too.
1. you can use structured mode with Avro too, right?
1. IIRC we wanted to default to structured mode and JSON encoding - is this also what's implemented? If it is, you should probably mention that here..

Reply:
1. I'll reword it
2,3. check `Producer details` and `Consumer details` they should provide enough information how it works -- if not, need change